### ISSUE TITLE
Request for root directory dependency graphs.

### ISSUE THREAD

**Issue by makelinux** [doxygen/doxygen#9391]
Currently directory dependency graphs are produced only for subdirectories for the project.
Why not project root directory?

Root directory dependency graph for Doxygen would look:

![Screenshot from 2022-06-11 15-04-56](https://user-images.githubusercontent.com/2335185/173187269-fd621e6f-8d0c-4c38-b2f2-ebd038492c1a.png)

**Comment by albert-github** [comment]
I think this is because the directory doxygen is not an `INPUT` directory for the doxygen internal documentation, the  setting is:
```
INPUT                  = src \
                         vhdlparser \
                         libxml
```
it is probably an trade-off between `INPUT` and `EXCLUDE...`

**Comment by makelinux** [comment]
Even with  `INPUT= .   RECURSIVE=yes`  

See example https://makelinux.gitlab.io/graphviz-doxygen/files.htm

**Comment by albert-github** [comment]
Yes as e.g. the directories like `addon`, `filesystem`, `liblodepng`, `libmd5`, `libmscgen`,  `libversion` .. and the file `README.md` are not in the internal documentation.
It is also historical motivated.
